templates/embed.lucius

```diff
@@ -7,8 +7,9 @@
 
 .carnival-comment-indicator {
   cursor: pointer;
-  position: absolute;
-  right: -30px;
+  float: right;
+  position: relative;
+  right: -15px;
```

Comment:
This is the only way I could find to do this without relying on the specifics of exactly where and how wide the user's `article` element is. It does cause some minor text-re-flow when the indicators get added to the DOM though. Unfortunately, I don't see a way to address this without re-working the entire approach (which I think we should do eventually).